### PR TITLE
Fix #75 design edits

### LINKED ISSUE #75: Design: Edits
- [x] #140
- [x] Edit introduction: 7.1 currently reads "It sets out recommended practices for documenting the Design as an Analytical Plan, making changes to the plan and keeping an audit trail of those changes, testing for reproducibility and uncertainty, and agreeing this with the Approver. An Analysis Steering Group can be an effective means for communications about the Design."
The rest of the chapter is sparse on details about "keeping an audit trail of those changes, testing for reproducibility ". Can we remove the entire sentence from the intro, or edit to match the most significant points covered in the chapter?
- [x] Edit "7.3 Testing the methodology": 
       This section covers a lot of points, and needs to be reworked.
      - "If the analysis is written in code, this should include appropriate testing to ensure that the code works as intended to deliver expected outcomes. Following [recommended good practices for code development](https://best-practice-and-impact.github.io/qa-of-code-guidance/intro.html) is the best way to ensure that code is fit for purpose." This confounds testing of the design with assurance of the analysis.
       - Do we want to change this to "Assuring the design"
       - Much of this paragraph does not speak to the subheading
- [x] Documenting the design:
       - Current: "Best practice is to document the design process recording how the proposed analytical process generates the requested insights. This plan should be supported by the design documentation." Change to "The design process should be documented in a proportionate manner. A design document that records the Analytical Plan should be produced by the Analyst, and signed-off by the commissioner"
       - If 7.4 is included then "Documentation of the Analytical Plan should be proportionate to the Design. For larger or more complex projects, the Analyst might request assurance of the Analytical Plan." can be removed from 7.2
       - Documenting the design should be added to Analyst responsibilities
- [x] Check that there is reference to ethics in design and add if needed
- [ ] Add information about "Assurance plan". The Analysis section reads "The analyst will also carry out the assurance plan and shall provide traceable documentation of the assurance they have undertake" This is the first mention of the "assurance plan" outwith the definitions. So we should add details of Assurance Plan to design section. Add hyperlinks from Analysis assurance plan to either first reference, or definition
- [x] Approver (and assurer) responsibilities repeated from Chapter 6 . Chapter 6:
"The assurer should understand the assurance requirements and plan time in for the assurance activity.
The Approver should document the new project and ensure that required mechanisms are in place for the appropriate assurance to take place. For example, they should ensure that the analyst and assurer are aware of local assurance protocols."
Chapter 7:
"The Assurer should agree with Analyst and Commissioner what the assurance requirements will be and plan sufficient time for the assurance activity.
The Approver should document the new project and ensure that required mechanisms are in place for the appropriate assurance to take place. For example, they should ensure that the Analyst and Assurer are aware of local assurance protocols"
- [ ] Production of analytical plan and assurance plan as outputs. These are referred to in Analysis and key concepts
- [x] Design considerations for multi-use models
- [x] Remove paragraph "The Commissioner remains important through the analysis cycle..." as not relevant to the design phase
- [x] Simplify materiality paragraph and move to "Designing the analysis"
- [x] Move paragraphs "On completion of the design phase.." and subsequent to "Designing the analysis"

### PR DESCRIPTION
**Pull request by irisoren-sg**

### CODE REVIEW

**Comment by irisoren-sg** [comment]
Apologies. Most the commits referred to the wrong issue. Should be #75

File: design.qmd [hunk 1]
```diff
@@ -33,58 +37,72 @@ The Analyst translates the scope for the analysis agreed with the Commissioner i
...
+
+
+ 
+
+
+
+
+## Treatement of uncertainty in the design phase
 
 During the design phase, Analysts should examine the planned analysis systematically for possible sources and types of uncertainty, to maximise the chance of identifying all that are sufficiently large to breach the acceptable margin of error.
```
Comment (Hurstharrier) [inline design.qmd:89]: Think we need to think about how we talk about what is an acceptable level of uncertainty. IN my experience most commissioners find specifying an acceptable level of uncertainty very difficult.

File: design.qmd [hunk 2]
```diff
@@ -33,58 +37,72 @@ The Analyst translates the scope for the analysis agreed with the Commissioner i
...
 
-The recommended approach for developing analysis in code is to use a [Reproducible Analytical Pipeline (RAP)](https://analysisfunction.civilservice.gov.uk/support/reproducible-analytical-pipelines/). Reproducible Analytical Pipelines shall:
 
-* Follow the practices set out in the [Analysis Function Quality Assurance of Code for Analysis and Research manual](https://best-practice-and-impact.github.io/qa-of-code-guidance/intro.html).
-* Meet the requirements of the [Reproducible Analytical Pipelines minimum viable product](https://github.com/best-practice-and-impact/rap_mvp_maturity_guidance/blob/master/Reproducible-Analytical-Pipelines-MVP.md).
 
-## Artificial intelligence and machine learning
+## Multi-use models and the design phase
+
+Designing multi-use models should take into account the needs of all users of the analysis. An Analysis Steering Group can be an effective means for communication about the design with a range of user groups.
```
Comment (Hurstharrier) [inline design.qmd:100]: think this should read "main users"

File: design.qmd [hunk 3]
```diff
@@ -33,58 +37,72 @@ The Analyst translates the scope for the analysis agreed with the Commissioner i
...
+AI and some classes of machine learning methods have opaque algorithms processing their inputs. These belong to a class of models known as “black boxes” - devices, systems, or objects which produce useful information without revealing any information about their internal workings.
 
-Using a black box model places greater weight on the design of the analysis and the assurance and validation of outputs by domain experts, particularly Assurers, Approvers and Commissioners.    [Validate AI](https://validateai.org/)  offers resources.
+Using a black box model places greater weight on the design of the analysis and the assurance and validation of outputs by domain experts, particularly Assurers, Approvers and Commissioners.    This [guidance](https://www.gov.uk/government/publications/introduction-to-ai-assurance/introduction-to-ai-assurance) outlines considerations for the design of AI models, including risk assessment, impact assessment, bias audits and compliance audits.
```
Comment (Hurstharrier) [inline design.qmd:108]: Think we agreed not to use this reference.

**Review by Hurstharrier — COMMENTED** [review]
@irisoren-sg  Have reviewed and made some edits and comments.

@rebecca-wodcke-gad  I think we need a more detailed reference on design and development of AI and ML models for the additional resources section

**Pull request merged**